config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/MoronsGuidetoBre-AAAAAAAAAAAAAAAAAAAIoQ==.json

```diff
@@ -31,7 +31,7 @@
bee that has little to no crossbreeding potential but has all the traits you want - short life, nocturnal, cave dwelling, rain tolerant, humidity tolerance, temperature tolerance, 4xfertility, and so on. Once you have that bee, breed a bunch of pure princesses with those traits as both active and inactive. These will be your breeding stock.\n\nWhen you are trying to get new species, what you want is a drone - this way you can use your \"breeding stock\" princess with the drone to get new species without the annoying effects or limitations. Poorly crossed princesses can be returned to normal by crossing with breeding stock drones, or just thrown into storage if they now have annoying restrictions. Make sure you keep the trait and species you want, and slowly merge the good traits from your breeding stock. You don\u0027t have to do this part in an Apiary, you can use a Bee House to swap around traits. Remember that no species mutations occur in a Bee House. Ignoble bees are good to swap traits because they\u0027re mostly disposable - they have a chance to die after 100 breeding cycles, increasing slightly more each generation after that, though genetic stability frames can let them live longer. \n\nOnce you solidify your species, create a pool of drones. You can use those to convert pristine princesses to the new species. Pristine princesses will not die, but they can become Ignoble under certain circumstances like using tons of production frames. Rocky bees are a great source - all their princesses are always pristine, they have decent traits, and there\u0027s a bunch of them underground. If you have an ender quarry, you can collect them with that.\n\nOnce you get a new species, it will replace either all Active or Inactive traits. If the mutation happens twice at the same bee, it will replace both sets.§r For example, when you get a Cultivated bee, either the Active or Inactive column will have all the default Cultivated traits. Mix it a few times and you\u0027ll eventually get purebred versions. Just watch out that you don\u0027t lose a key trait, like Fast Worker for Cultivated. Most effects are useless, so you can disregard them.\n\nAll quests require purebred§r species. That means that the Inactive species-trait should be the same as the Active one.\n\nAll traits can be bred into any bee species, aside from the species itself. Species also determines the preferred temperature and humidity, as well as drops. Species require their preferred temperature and humidity to be in the \u0027jubilant\u0027 state and produce their special drops. You can also mimic this with the alveary or industrial apiary, but NOT with tolerances.\n\nAdvanced frames can provide useful boosts to mutation chances or lifespan reduction for faster breeding cycles. However, the excessive multipliers will increase genetic decay, giving a chance for a pristine princess to become ignoble.\n\nIt looks complex, because it is! But bees provide a lot of convenience at a distance."
```

Comment:
I rewrote it because it was hard to understand in many places. 

If my rewrite is lacking, I'll do my best to improve it - reverting the whole thing back won't fix it.

Reply:
> you dont need both

You're correct, one doesn't need both. I described one of the three cases, the case dominant-recessive. The other two are dominant-dominant and recessive-recessive. Or did you mean something else?

Reply:
Hopefully fixed. Applied all of the suggestions in one form or the other.

Reply:
the dominant/recessive is not fixed. its not about breeding, just having about having them. and it still sounds like you dominant only beats recessive.